src/index.html

```diff
@@ -7,8 +7,72 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.cdnfonts.com/css/roboto?styles=14400" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="#logo">
+          <img class="img" src="./images/logo.png"
+          alt="img">
```

Comment:
```suggestion
          <img 
            class="img" src="./images/logo.png"
            alt="img"
          >
```
Consider fixing indentations according to the code style guide